pex/requirements.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, filename):
 
 # Process lines in the requirements.txt format as defined here:
 # https://pip.pypa.io/en/latest/reference/pip_install.html#requirements-file-format
+# Note that the lines can be not just pip-style requirement specifiers as detailed
+# at the link above, but any string representation of a Resolvable that pex understands,
+# including package URLs and setup.py directories.
```

Comment:
Your comment is true, so lgtm, but, as of today, Resolvable.get is still a strict subset. You can pip install an url pointing at a package as well as a local directory containing a setup.py.

Reply:
Interesting. That isn't mentioned in that pip documentation though.

Reply:
It sortof is. The archive utl/path one is explicit. The local project path less so, since they only talk about the editable (prefix with -e) case. You can install a local project without -e and it just never updates after that if edited.

---
Each line of the requirements file indicates something to be installed, and like arguments to pip install, the following forms are supported:
```
[[--option]...]
<requirement specifier> [; markers] [[--option]...]
<archive url/path>
[-e] <local project path>
[-e] <vcs project url>
```
For details on requirement specifiers, see Requirement Specifiers.